addons/mail/models/res_users.py

```diff
@@ -18,6 +18,16 @@ class ResUsers(models.Model):
     """
     _inherit = 'res.users'
 
+    res_role_ids = fields.Many2many(
+        "res.role",
+        string="Res Roles",
```

Comment:
This name makes no sense, we should not have Res in user facing strings.

I really think it should be named Roles and remove the failing test.
Or worse case if we must keep it, User Roles.

Reply:
Or "User Roles".
Sure this sucks that there's this constraint and a specific localization reserved this "Roles" string, but there are easy workarounds like that that are not that bad IMO.